inc/markup-extras.php

```diff
@@ -187,9 +187,9 @@ function astra_logo( $device = 'desktop', $echo = true ) {
 		$site_title           = astra_get_option( 'display-site-title-responsive' );
 		$display_site_title   = ( $site_title['desktop'] || $site_title['tablet'] || $site_title['mobile'] ) ? true : false;
 
-		$html = '';
-
+		$html            = '';
 		$has_custom_logo = apply_filters( 'astra_has_custom_logo', has_custom_logo() );
+		do_action( 'astra_before_logo_image', $device );
```

Comment:
```suggestion
		do_action( 'astra_before_logo_image', $device );
```
@imnavanath Any reason we are introducing this action?

Reply:
We added this new action to insert dark mode logo which will come from addon, you can ref that here - https://github.com/brainstormforce/astra-addon/blob/pro-dark-mode-features/addons/dark-mode-switch/classes/class-astra-addon-dark-mode-markup.php#L68